pymess/management/commands/pull_emails_info.py

```diff
@@ -19,7 +23,13 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         email_sender = get_email_sender()
 
-        messages = EmailMessage.objects.filter(require_pull_info=True).order_by('-sent_at')
+        delay = datetime.timedelta(seconds=settings.EMAIL_PULL_INFO_DELAY_SECONDS)
+        messages = EmailMessage.objects.filter(
+            Q(info_changed_at__isnull=True) | Q(info_changed_at__lt=F('last_webhook_received_at') + delay),
+            last_webhook_received_at__isnull=False,
+            last_webhook_received_at__lte=now() + delay,
```

Comment:
tohle je tam svým způsobem 2x

Reply:
No hlavně tahle podmínka je špatně :-1: 

Má to být:

`last_webhook_received_at__lte=now() - delay`